bay/plugins/run.py

```diff
@@ -77,25 +80,50 @@ def run(app, containers, host, tail):
 @click.command()
 @click.argument("container", type=ContainerType())
 @click.option("--host", "-h", type=HostType(), default="default")
+@click.option("--running", "-r", is_flag=True)
+@click.option("--not-running", "-n", is_flag=True)
```

Comment:
these two definitely need help strings, because I find them confusing even knowing the code.